src/Components/InstanceCounter/index.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { Slider } from '@patternfly/react-core';
 import { useWizardContext } from '../Common/WizardContext';
+import './styles.css';
```

Comment:
This import works well, it just wraps the css classes with a `provisioning` class
For example, if we define a new class `.example` it built as
```css
.provisioning .example {
  width: auto
}

therefore I added a `provisioning` class to our wizard down below